src/main/java/uk/gov/companieshouse/company/profile/converter/JsonDateTimeConverter.java

```diff
@@ -25,6 +30,7 @@ public void convert(Long value, StrictJsonWriter writer) {
             String formattedDate = DATE_TIME_FORMATTER.format(instant);
             writer.writeString(formattedDate);
         } catch (Exception ex) {
+            LOGGER.error(ex.getMessage());
```

Comment:
Can you please add a meaningful message in front of the exception?

Reply:
Yes, sure